tuf_conformance/conftest.py

```diff
@@ -83,14 +77,25 @@ def static_client(
     return ClientRunner(entrypoint, static_server, request.node.name)
 
 
+@cache
+def read_xfails(pytestconfig: pytest.Config) -> list[str]:
+    # Find expected failures from .xfails file
+    xfail_file = f"{pytestconfig.getoption('--entrypoint')}.xfails"
+    if not os.path.isabs(xfail_file):
+        xfail_file = os.path.join(pytestconfig.invocation_params.dir, xfail_file)
+
+    try:
+        with open(xfail_file) as f:
+            return f.read().splitlines()
```

Comment:
it might be useful to parse the content a bit -- e.g. allow comments so you could do something like this

```
# RSA PSS not yet supported https://github.com/theupdateframework/go-tuf/pull/625 
test_keytype_and_scheme[rsa/rsassa-pss-sha256]
```
in the .xfails file... but maybe we add that when someone actually wants it